etc/docker/kernel-tf-gpu-py/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Ubuntu:xenial
-ARG BASE_CONTAINER=tensorflow/tensorflow:1.12.0-gpu-py3
+ARG BASE_CONTAINER=tensorflow/tensorflow:2.4.0-gpu-py3
```

Comment:
This tag doesn't exist.  There's `2.4.0-gpu` and `2.4.0-gpu-jupyter` as well as another: https://hub.docker.com/r/tensorflow/tensorflow/tags?page=1&ordering=last_updated&name=2.4.0.